google-cloud-bigquery/pom.xml

```diff
@@ -12,7 +12,7 @@
   <parent>
     <groupId>com.google.cloud</groupId>
     <artifactId>google-cloud-pom</artifactId>
-    <version>0.8.1-alpha</version>
+    <version>0.8.2-alpha-SNAPSHOT</version>
```

Comment:
Since we aren't going to have a `0.8.2` version, only a `0.8.2-alpha` version, `0.8.2-alpha-snapshot` seems the most correct.

This tagging of the release status in the version number is the root problem. It really is too bad we can't use a development status classifier like Python does instead of mucking with version numbers. https://packaging.python.org/distributing/#classifiers